docs/path-for-nested-resource.md

```diff
@@ -20,6 +20,10 @@ The path for nested resource doest not meet the valid resource pattern.
 
 Path for CRUD methods on a nested resource type MUST follow valid resource naming, like '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MyNameSpace/MyResourceType/{Name}/NestedResourceType/{nestedResourceName}'.
 
+There is one exception where extension resources with fully qualified path of the below format can exist
+"/providers/Microsoft.Management/managementGroups/{mgId}/{resourceProviderScope}/providers/Microsoft.Quota/groupQuotas/{groupQuotaName}/groupQuotaRequests/{requestId}"
```

Comment:
>Microsoft.Management/managementGroups [](http://example.com/codeflow?start=12&length=37)

lets give a different example instead of managementgroups. ManagementGroups is a special RP and has some issues with this pattern.